src/optimizer/plugins/ResolveValueRuleURLsOptimizer.ts

```diff
@@ -1,21 +1,21 @@
-import { fshtypes } from 'fsh-sushi';
+import { fshtypes, utils } from 'fsh-sushi';
 import { OptimizerPlugin } from '../OptimizerPlugin';
 import { Package } from '../../processor';
 import {
   ExportableAssignmentRule,
   ExportableCaretValueRule,
   ExportableRule
 } from '../../exportable';
-import { resolveAliasFromURL } from '../utils';
+import { optimizeURL } from '../utils';
 import CombineCodingAndQuantityValuesOptimizer from './CombineCodingAndQuantityValuesOptimizer';
-import { ProcessingOptions } from '../../utils';
+import { ProcessingOptions, MasterFisher } from '../../utils';
 
 export default {
   name: 'resolve_value_rule_urls',
   description: 'Replace URLs in caret and assignment rules with their names or aliases',
```

Comment:
This isn't the fault of this PR, but... this optimizer seems to be poorly named / described, because it's much narrower than it suggests. It doesn't resolve all value rule URLs; it only resolves the system URLs of codes in value rules. It may be more accurate to call it `resolve_value_rule_system_system_urls_for_codes` (I almost said `resolve_value_rule_code_system_urls`, but that can also be misunderstood).